monkestation/code/modules/admin/smites/rip_and_tear.dm

```diff
@@ -1,17 +1,17 @@
-/datum/smite/rip_and_tear

```

Comment:
me when CRLF

(git always automatically changes the line endings of this file from CRLF to LF, every time I `git checkout` or anything)